hw/ip/lc_ctrl/lc_ctrl.core

```diff
@@ -17,11 +17,11 @@ filesets:
       - lowrisc:prim:sparse_fsm
       - lowrisc:ip:lc_ctrl_pkg
       - lowrisc:ip:tlul
-      - lowrisc:ip_interfaces:pwrmgr_pkg
+      - lowrisc:virtual_ip:pwrmgr_pkg
```

Comment:
Just a note: This one is a good candidate for dropping by moving the related definitions to lc_ctrl (or some common lib of interfaces). The references don't depend on any ipgen template parameters.

Reply:
Yes, I intend to deal with this in another PR.